spec/formatting.md

```diff
@@ -259,6 +259,12 @@ The resolution of a _variable_ fails if no value is identified for its _name_.
 If this happens, an _Unresolved Variable_ error is emitted
 and a _fallback value_ is used as the _resolved value_ of the _variable_.
 
+If the _resolved value_ identified for the _variable_ _name_ is a _fallback value_,
+a _fallback value_ is used as the _resolved value_ of the _variable_.
```

Comment:
A general issue: The _resolved value_ is logically {datatype, optionMap}. So whenever we have a statement like this, that implies that a _fallback value_ is a _resolved value_. That means we **must** say what the optionMap is: I presume an empty map, but we need to be clear.

Reply:
Good point. I've updated the text to make the references more explicit. PTAL?

Reply:
Excellent fixes to all the fallback value text, thanks!

Reply:
This sentence reads as a tautology. Perhaps:

```suggestion
If the _resolved value_ associated with a _variable_ _name_ is a _fallback value_,
then the _resolved value_ of the _variable_ is that _fallback value_.
```

Reply:
That's not quite right, though, because the fallback value needs to be different. Consider here:
```
.local $x = {42 :func-fails}
.local $y = {$x}
{{{$x}, {$y}}}
```
The above should format as "{$x}, {$y}", and that requires for the `$y` fallback value to be different from the `$x` fallback value.